pom.xml

```diff
@@ -318,23 +318,15 @@
 							<goal>prepare-agent</goal>
 						</goals>
 						<configuration>
-							<!-- Sets the path to the file which contains the execution data. -->
-							<destFile>${sonar.jacoco.reports}/jacoco-it.exec</destFile>
 							<propertyName>failsafeArgLine</propertyName>
```

Comment:
In other POMs this uses placeholder syntax like
```
<propertyName>failsafeArgLine</propertyName>
```
Can you check which is correct?

Reply:
This seems to be consistent across other poms

Reply:
metrics-consumer uses these property names also